lib/bundix.rb

```diff
@@ -68,6 +68,7 @@ def groups(spec, dep_cache)
     "mingw" => [{engine: "mingw"}],
     "truffleruby" => [{engine: "ruby"}],
     "x64_mingw" => [{engine: "mingw"}],
+    "windows" => [{engine: "mswin"}, {engine: "mswin64"}, {engine: "mingw"}],
```

Comment:
Wasn't 100% sure what `engines` to list here, but basically took the list from `Gem::Platform::WINDOWS` as defined here

https://github.com/rubygems/rubygems/blob/8c98dbf1b1f6005b4e0343999d7fd556428fa0c2/bundler/lib/bundler/rubygems_ext.rb#L208